anonymizer/OpenDiffix.Service/AggregationHooks.fs

```diff
@@ -148,11 +148,10 @@ module StarBucket =
   let private makeStarBucket (aggregationContext: AggregationContext) =
     let isGlobal = isGlobalAggregation aggregationContext
 
-    // TODO: Do we need a specific noise seed for the star bucket?
     let executionContext = aggregationContext.ExecutionContext
```

Comment:
I would get rid of this local binding because there's a chance we might accidentally use `aggregationContext.ExecutionContext` instead of `bucket.ExecutionContext`.

Reply:
Ah, I see that this is a short function. I thought it spans all the way below.